backend/gs_data_dir/workspaces/decsiraogc_controlli_scarichi_urbani/decsiraogc/ControlliScarichiUrbani/featuretype.xml

```diff
@@ -0,0 +1,67 @@
+<featureType>
+  <id>FeatureTypeInfoImpl--40547dcb:16ea733684b:-7e45</id>
+  <name>ControlliScarichiUrbani</name>
+  <nativeName>decsira_geo_controlli_scarichi_urbani</nativeName>
```

Comment:
Dovresti impostare anche il Title, con valore "Controlli Scarichi Acque Reflue"

Reply:
Il title col nome richiesto è definito più in basso nel file